esmvaltool/diag_scripts/carbon_cycle/main.ncl

```diff
@@ -0,0 +1,626 @@
+; #############################################################################
+; carbon_cycle/main.ncl
+; #############################################################################
+; DIAGNOSTIC SCRIPT FOR ERRORBAR AND SEASONAL CYCLE PLOTS
+; Author: Alessandro Anav (University of Exeter, UK)
+; EMBRACE project
+; #############################################################################
+;
+; Description:
+;     Calculates temporal and spatial averages and plots the variable as
+;     error-bar and seasonal cycle plots.
+;
+; Required diag_script_info attributes:
+;     region: the region to be averaged.
+;     legend_outside: plot legend in a separate page (not for errorbar plot).
+;     seasonal_cycle_plot: draw seasonal cycle plot [False/True].
+;     errorbar_plot: draw errorbar plot [False/True].
+;     mean_IAV_plot: draw Mean (x-axis), IAV (y-axis) plot [False/True].
+;     evolution_plot: time evolution of a variable comparing obs to
+;                     multi-dataset mean; requires ref_dataset in recipe and
+;                     at least one model and one observation.
+;
+; Optional diag_script_info attributes:
+;     sort: sort dataset in alphabetical order.
+;     anav_month: Conversion of y-axis to PgC/month instead of /year,
+;                 following Anav2013 fig 7, 9 (cycle plot).
+;     evolution_plot_ref_dataset: reference dataset for evolution_plot.
+;     evolution_plot_anomaly: makes evolution_plot an anomaly plot.
+;     evolution_plot_ignore: Datasets to ignore, obs are excluded
+;                            automatically.
+;     evolution_plot_volcanoes: Turns on/off lines of volcano eruptions.
+;     evolution_plot_color: Hue of the contours; default: red = 0.
+;
+;     For external dataset input:
+;     obsfile: Filename of external obs to read in.
+;     obsname: Name of ext dataset to read in.
+;     ref_varname: Varname in attributes of ext file.
+;
+; Caveats:
+;     If CRU is the reference dataset it is important to apply a landmask in
+;     the preprocessor section of the recipe.
+;     Regridding is not required since spatial averages are calculated for all
+;     plots. Performance metrics calculated with perfmetrics_main could be
+;     inconsistent with the plots produced by this script, since a consistent
+;     missing value mask is not applied here. However, for variable such NBP,
+;     GPP, LAI, FGCO2 for which this script is designed, there should be no
+;     inconsistencies as the missing value mask is similar for all datasets
+;     and observations and coincides with the land-sea mask.
+;
+; Modification history:
```

Comment:
Please change the author tags according to the new standard (e.g, `schl_ma` --> `schlund_manuel`).